swagger.yaml

```diff
@@ -1,75 +1,392 @@
+      managerUid:
+        type: string
+        description: UID of the managing person.
+      managingGroupID:
+        type: string
+        description: ID of the managing group.
+      managingGroupName:
+        type: string
+        description: name of the managing group.
       name:
         type: string
-        description: Name of the API.
-      time:
+        description: service name.
+      newTicketUrl:
         type: string
-        description: Build time in ISO 8601 format.
-      unixTime:
+        format: url
+        description: new ticket URL
+      order:
         type: integer
-        description: Build time as milliseconds since the unix epoch.
-      commit:
+        description: order of the service among its siblings.
+      requestApplicationID:
         type: string
-        description: Git commit hash of the revision which was built.
-      documentation:
+        description: ID of the destination application for any generated requests.
+      requestApplicationIsActive:
+        type: boolean
+        description: active status of the destination application for any generated requests.
+      requestApplicationName:
         type: string
-        description: Filename of the swagger specification for the API.
-  Error:
+        description: name of the destination application for any generated requests.
+      requestText:
+        type: string
+        description: custom text used for the "Request This" button.
+      requestTypeCategoryID:
+        type: string
+        description: ID of the associated request type's containing category.
+      requestTypeCategoryName:
+        type: string
+        description: name of the associated request type's containing category.
+      requestTypeComponent:
+        type: integer
+        enum: [0, 1, 4, 9]
+        description: >
+          Sort oder:
+            * 0  - [None] Indicates that a service is not tied to any particular kind of request.
+            * 1  - [Project] Indicates that a service is tied to some type of project request.
+            * 4  - [Link] Indicates that a service is tied to an external request available at a custom URL.
+            * 9  - [Ticket] Indicates that a service is tied to some sort of ticket request.
+      requestTypeID:
+        type: string
+        description: ID of the associated request type.
+      requestTypeIsActive:
+        type: boolean
+        description: a boolean value indicating whether the associated request type is active.
+      requestTypeName:
+        type: string
+        description: name of the associated request type.
+      requestUrl:
+        type: string
+        format: url
+        description: custom request URL configured for this service.
+      shortDescription:
+        type: string
+        description: service's short description, which is used when displaying the service in a list
+      spanTagsParsedFromLongDescription:
```

Comment:
Since we're making some changes to the response structure, what do you think about collapsing these fields to be in the same attributes object as the other fields? That way, attributes doesn't have any subobjects.